.github/workflows/check_dependency_duplications.yml

```
@@ -1,39 +1,65 @@
-name: Check npm dependency duplications
-
+name: "npm-dependency-duplications test"
 on:
-  workflow_call:
-    inputs:
-      node-version:
-        default: 16
-        description: Node version passed to the actions/setup-node
-        required: false
-        type: string
-      cache:
-        default: npm
-        description: Package manager for caching - passed to the actions/setup-node
-        required: false
-        type: string
-      project-path:
-        default: .
-        description: NPM project localization
-        required: false
-        type: string
+  pull_request:
+  push:
+    branches:
+      - master
+    paths:
+      - 'npm-dependency-duplications'
+  workflow_dispatch:
+
 
 jobs:
-  check_duplications:
-    name: Check dependency duplications
+  units:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set Node.js 16
```

Same suggestion for other places too:
```suggestion
      - name: Set up Node.js 16
```